homeassistant/components/zwave_js/helpers.py

```diff
@@ -66,6 +66,19 @@ def get_device_id(client: ZwaveClient, node: ZwaveNode) -> tuple[str, str]:
     return (DOMAIN, f"{client.driver.controller.home_id}-{node.node_id}")
 
 
+@callback
+def get_device_id_ext(client: ZwaveClient, node: ZwaveNode) -> tuple[str, str] | None:
+    """Get extended device registry identifier for Z-Wave node."""
+    if None in (node.manufacturer_id, node.product_type, node.product_id):
+        return None
+
+    domain, dev_id = get_device_id(client, node)
+    return (
+        domain,
+        f"{dev_id}-{node.manufacturer_id}:{node.product_type}:{node.product_id}",
```

Comment:
are these fields available early ?
If a node is being added these will still be empty, right ?

Reply:
sorry, I should have read more carefully. You're adding this identifier as an EXTRA identifier to the device.
Perfect!

Reply:
Correct on both counts, when the node is first added they are not available, and later after the interview and the node is ready, they are being used as extra ids.